src/main/java/org/opensearch/security/dlic/rest/api/FlushCacheApiAction.java

```diff
@@ -43,16 +43,18 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import org.opensearch.security.securityconf.impl.CType;
-
 import com.google.common.collect.ImmutableList;
+import static org.opensearch.security.dlic.rest.support.Utils.addRoutesPrefix;
```

Comment:
`Utils.addRoutesPrefix.addRoutesPrefix` is imported twice, please fix. Separate static import from other imports by an empty line.

Reply:
Empty line needs to be added **before** static import, not after. Please take as an action item for follow up PR.